### ISSUE TITLE
any documentation?

### ISSUE THREAD

**Issue by yathit** [yradtsevich/pure-js-websql#1]
100% websql implementation is interesting. any documentation for supported operations?

**Comment by yradtsevich** [comment]
@yathit it supports virtually everything what Chrome WebSQL implementation supports. So almost any code for Chrome will work with this implementation.

I know only two very minor differences from Chrome:
- There are no security filter (for example SQLITE_CREATE_TABLE will work, but must not)
- SELECT always performs type conversion based on the returned value (e.g. SQL Text "1" will be automatically converted into 1.0 (JavaScript Number type)). This means you should prefer '==' over '===', especially for Text fields.

**Comment by yathit** [comment]
How about JOIN and ORDER BY ?

**Comment by yradtsevich** [comment]
Yes, everything like this is supported :)

**Comment by yathit** [comment]
really? I don't see those in test file or even in the implementation. Am I looking for wrong place?

**Comment by yradtsevich** [comment]
@yathit Yes! This is not a joke ;) This library wraps [SQL.js](https://github.com/kripken/sql.js) - an awesome SQLite port to JavaScript, which does all underlying work.
test.html [links to it](https://github.com/yradtsevich/pure-js-websql/blob/master/test/index.html#L4).

I just updated [demo page](http://yradtsevich.github.io/pure-js-websql/test/index.html) with a more complex example.

**Comment by yathit** [comment]
Awesome! Emscripten things. 

I really wish to have SQL processor, on top of IndexedDB. localStorage still usable, Great work! Even on memory storage is still useful.

Thanks,
Kyaw
